R4ML/R/ml.model.lm.R

```diff
@@ -338,10 +339,127 @@ predict.r4ml.lm <- function(object, data) {
 
     # orgainze result for presentation to user
     if (testing) {
-      stats <- SparkR::as.data.frame(r4ml.read.csv(statsPath, header=FALSE, stringsAsFactors=FALSE))
-      return(list("predictions"=preds, "statistics"=stats))
+      stats <- SparkR::as.data.frame(r4ml.read.csv(statsPath, header = FALSE,
+                                                   stringsAsFactors = FALSE))
+      return(list("predictions" = preds, "statistics" = stats))
+    } else {
+      return(list("predictions" = preds))
     }
-    else {
-      return(list("predictions"=preds))
+}
+
+r4ml.lm.se <- function(object, seed = 5) {
+  logSource <- "r4ml.lm.se"
+
+  df <- object@training_data
+  training_nrow <- SparkR::nrow(df)
+
+  if (training_nrow > 100000) {
+    # since the dataset has >100k rows we want a sample we can fit on the driver
+    frac <- 100000 / training_nrow # we want a sample of ~50k rows
+    df <- SparkR::sample(df, withReplacement = FALSE, fraction = frac, seed = seed)
+  }
+
+  df <- SparkR::as.data.frame(df)
+
+  colnames(df)[which(colnames(df) == object@labelColumnName)] <- "y"
+
+  if (object@intercept) {
+    fit <- lm(y ~ ., data = df )
+  } else {
+    fit <- lm(y ~ . -1, data = df )
+  }
+
+  coefs <- summary(fit)$coefficients
+  coefs <- as.data.frame(coefs)
+
```

Comment:
I disagree. I think having less code makes it less complicated and more readable